CLAUDE.md

```diff
@@ -0,0 +1,141 @@
+# CLAUDE.md
+
+This file provides guidance to Claude Code (claude.ai/code) when working with code in this repository.

```

Comment:
This sentence looks a bit out-of-sync with the content, at least how I understand it.